user-interface/src/main/java/life/qbic/datamanager/security/UserPermissionsImpl.java

```diff
@@ -12,6 +12,8 @@ public class UserPermissionsImpl implements UserPermissions {
 
   final AclPermissionEvaluator aclPermissionEvaluator;
 
+  private static final String projectTargetType = "life.qbic.projectmanagement.domain.model.project.Project";
+
```

Comment:
You can replace the hardcoded string name by using one of the java name methods for the class of interest outlined here:
https://stackoverflow.com/questions/15202997/what-is-the-difference-between-canonical-name-simple-name-and-class-name-in-jav